api/v1alpha1/perconaservermysqlbackup_types.go

```diff
@@ -18,25 +18,33 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // PerconaServerMySQLBackupSpec defines the desired state of PerconaServerMySQLBackup
 type PerconaServerMySQLBackupSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of PerconaServerMySQLBackup. Edit perconaservermysqlbackup_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	ClusterName string `json:"clusterName"`
+	StorageName string `json:"storageName"`
 }
 
+type BackupState string
```

Comment:
We need to have the same status for ps operator 
```
❯ kubectl get pxc-backup
NAME                           CLUSTER         STORAGE   DESTINATION                                                    STATUS      COMPLETED   AGE
on-demand-backup-aws-s3        demand-backup   aws-s3    s3://operator-testing/demand-backup-2022-06-10-12:33:46-full   Running                 24s
on-demand-backup-pvc           demand-backup   pvc       pvc/xb-on-demand-backup-pvc                                    Succeeded   14m         15m
on-demand-backup-pvc-bsource   demand-backup   pvc       pvc/xb-on-demand-backup-pvc-bsource                            Succeeded   7m8s        7m52s
```

It is very important to have `DESTINATION`

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/2d0f73080e2fc8d5794437d9069660bbd5547a3e